File: cdk-from-cfn-testing/expected/bucket/csharp/BucketStack.cs
using Amazon.CDK;
using Amazon.CDK.AWS.S3;
using Constructs;
using System.Collections.Generic;

namespace BucketStack
{
    public class BucketStackProps : StackProps
    {
    }

    public class BucketStack : Stack
    {
        public BucketStack(Construct scope, string id, BucketStackProps props = null) : base(scope, id, props)
        {

            // Resources
            var bucket = new CfnBucket(this, "Bucket", new CfnBucketProps
            {
            });
        }
    }
}

File: cdk-from-cfn-testing/expected/bucket/typescript/BucketStack.ts
import * as cdk from 'aws-cdk-lib';
import * as s3 from 'aws-cdk-lib/aws-s3';

export interface BucketStackProps extends cdk.StackProps {
}

export class BucketStack extends cdk.Stack {
  public constructor(scope: cdk.App, id: string, props: BucketStackProps = {}) {
    super(scope, id, props);

    // Resources
    const bucket = new s3.CfnBucket(this, 'Bucket', {
    });
  }
}
